ofonotextchannel.cpp

```diff
@@ -522,8 +522,9 @@ void oFonoTextChannel::mmsReceived(const QString &id, uint handle, const QVarian
 QDateTime oFonoTextChannel::getSentDate(const QString &sentTime){
     QDateTime dt = QDateTime::fromString(sentTime, Qt::ISODate);
     QDateTime currentDate  = QDateTime::currentDateTimeUtc();
-    //some text message may not have the Sentime set, use the received one in that case
-    if (!dt.isValid() || dt > currentDate){
+    // some text message may not have the Sentime set or having a 0 unix timestamp, use the received one in that case
+    // assuming that a sentdate less than four months ago is not a valid date
```

Comment:
What is the rationale for the 4 months value here? Also, the correct English for the comment should be "more than four months ago" or "prior to 120 days before the current time" for example, as "less than four months ago" would be newer than then (between four months ago and now).

Reply:
no rationale here;-), i. was just thinking of a time period where someone can be disconnected but i bet the sms will be deleted by the provider before

Reply:
btw, the date issue seems to be fixed, waiting for issue flagged as resolved before closing this PR